etc/containerpilot.json

```diff
@@ -1,16 +1,26 @@
 {
   "consul": "{{ .CONSUL }}:8500",
-  "preStart": [
-    "consul-template", "-once", "-consul", "{{ .CONSUL }}:8500", "-template",
-    "/etc/prometheus/prometheus.yml.ctmpl:/etc/prometheus/prometheus.yml"
-  ],
+  "preStart": "prestart.sh",
   "services": [
     {
       "name": "prometheus",
       "port": 9090,
-      "health": ["curl", "-so", "/dev/null", "http://localhost:9090/metrics"],
+      "health": ["curl", "-fso", "/dev/null", "http://localhost:9090/metrics"],
       "poll": 10,
       "ttl": 25
     }
+  ],
+  "coprocesses": [{{ if .CONSUL_AGENT }}
+    {
+      "name": "consul-agent (host:{{ .CONSUL }})",
+      "command": ["/usr/local/bin/consul", "agent",
+                  "-data-dir=/var/lib/consul",
+                  "-config-dir=/etc/consul",
```

Comment:
The alignment of these two dirs with the dirs created in the Dockerfile is frustratingly important.